bin/lint-commit-message

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+set -euf -o pipefail
+
+# Find the number of the most recently opened pull request merged so far.
+# Add 1 to this to get the smallest possible number of the next pull request,
+# then add an arbitrary buffer to allow for issues and pull requests opened
+# since the creation of the most recently opened pull request merged so far.
+# This provides an estimate of the number that will appear in the summary of
+# the merge commit so its length can be taken into account.
+
+pull_request_number() {
+  {
+    git log --grep '^Merge pull request #[1-9]' --pretty=format:%s \
+    | cut -d '#' -f 2 \
+    | cut -d ' ' -f 1 \
+    | sort --numeric-sort --reverse ;
+    echo 0 ;
+  } \
+  | head -n 1 \
+  | xargs expr 1 + 25 +
```

Comment:
Rather than adding an arbitrary buffer to the PR *number*, can't we just add a single *character* buffer? So an additional reserved character to allow for the expansion of the PR number? That would mean that:

After the **1st** PR, we reserve **9** slots
Once we reach **10** PR's, we reserve **990** slots
Once we reach **100** PR's, we reserve **9900** slots
...etc...

To me this approach seems less arbitrary, and more scalable. On the other hand, it might be overkill, and it may be upsetting to have 90% of all merge commit messages needlessly limited to -1 characters.